utils/prometheus-node-exporter-lua/files/usr/lib/lua/prometheus-collectors/realtek-poe.lua

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env lua
```

Comment:
```suggestion
```
This is not needed

Reply:
Oh right because `prometheus-node-exporter-lua` actually imports the collectors as libraries (or the `lua` equivalent of that) instead of executing it (which totally makes sense because I have not imported anything related to prometheus but can easily call `metric`).

Thanks for the hint :)